### ISSUE TITLE
Image doesn't show after selecting from computer

### ISSUE THREAD

**Issue by Swapnil2121993** [damonbauer/react-cloudinary#1]
No error shown. But when I select image from my computer nothing is showing on the box.
Here is my code :

import React,{Component} from 'react';
import Dropzone from 'react-dropzone';
import request from 'superagent';

const Upload_Preset="g3hmcumi";
const URL="https://api.cloudinary.com/v1_1/dhcnghcuh";

 export default class Profile extends Component{
  constructor(props){
  super(props);
  this.state={
    uploadedFile:null,
    url:'',
  };
  // this.onImageDrop=this.onImageDrop.bind(this);
}
onImageDrop = (files) => {
  this.setState({uploadedFile:files[0]});
  this.handleUpload(files[0]);
  }

handleUpload = (file) => {
  let upload = request.post(URL)
                      .field('Upload_Preset',Upload_Preset)
                      .field('file',file);
  upload.end = (error,response) =>{
      if(error){
        console.error(error);
      }

      if(response.body.secure_url !==''){
        this.setState({url:response.body.secure_url});
      }
  }
}
render(){
  return(
        <div>

            <Dropzone
              multiple={false}
              accept="image/*"
              onDrop={this.onImageDrop.bind(this)}
            >
              <p> Drop or click on select to upload image </p>
            </Dropzone>

          <div>
         {this.state.url === '' ? null :
         <div>
           <p>{this.state.uploadedFile.name}</p>
           <img src={this.state.url} />
         </div>}
       </div>
</div>

  );
}

}

**Comment by damonbauer** [comment]
@Swapnil2121993 - it looks like your upload url is incorrect.

It needs to match the URL here (substitute `react-cloudinary` for your cloudinary project name): https://github.com/damonbauer/react-cloudinary/blob/master/src/App.js#L7

**Comment by Swapnil2121993** [comment]
I am using image upload as a part of my application. My project name is
angular-react so you want me to change my upload url as
https://api.cloudinary.com/v1_1/angular-react/upload' ? I did this way but
still not working . If I write my url this way than where can I specify my
cloudinary cloud-name like
API Base URL:
https://api.cloudinary.com/v1_1/dhcnghcuh
▼

**Comment by damonbauer** [comment]
Did you try `'https://api.cloudinary.com/v1_1/dhcnghcuh/upload'`?

**Comment by Swapnil2121993** [comment]
yes i did. when you upload picture it should show you some request right in
network part as we are using request.post but in my case it does't show any
request. When i select image and click open nothing comes out in drop box.
You can see in screenshot
